### ISSUE TITLE
Report on local changes

### ISSUE THREAD

**Issue by saberraz** [lutraconsulting/input#1823]
It will be useful for the surveyors to be able to see the list of local changes. We have a feature already showing local changes but it is very hidden and not very descriptive.

I suggest to have a Survey Status (or something similar) to pull up a report on:

**Changes since DATE** (time of the last sync)
- Layers edited: xxx
   - Features added: xxx features
   - Features deleted: xxx features
   - Features modified: xxx features
   - Photos added

**Comment by rjwillson** [comment]
yes this would be an excellent feature, including the ability to email the report. 

I am often asked by users in the field if I can see their data in mergin or QGIS after they have synced while out in the field or after returning to their home office. A report like this would allow me to say "yes, the number of records in your report matches my mergin report and what I see here in QGIS.
